nautobot/project-static/css/hide_body.css

```diff
@@ -0,0 +1,5 @@
+body {
```

Comment:
Maybe add this to `base.css` instead of creating a separate stylesheet? Also please add a comment explaining why/how this is used.

Reply:
I'll add a comment for sure.
However this CSS code cannot go into `base.css`.  It's purpose is to hide the contents of `<body>` until after the Javascript event `DOMContentLoaded` is triggered, which occurs after the page is fully loaded.  Then as part of the JS, `<body>` is displayed.  This is what helps with the white screen flashing issue.

Reply:
OK, thanks for the explanation!